### ISSUE TITLE
Use semantic-release for fully-automated releases

### ISSUE THREAD

**Issue by davidyuk** [aeternity/aepp-sdk-js#1104]
https://github.com/semantic-release/semantic-release

**Comment by subhod-i** [comment]
@davidyuk Is this still on?

**Comment by davidyuk** [comment]
Yep, currently, we are using `standard-version` that is less automated.

https://github.com/conventional-changelog/standard-version#how-is-standard-version-different-from-semantic-release

https://www.npmjs.com/package/semantic-release vs https://www.npmjs.com/package/standard-version
`semantic-release` seems to be more popular, interesting to try this, but we may have problems with last-minute fixes like fixing formatting of the changelog
